mne/export/_edf.py

```diff
@@ -208,19 +235,17 @@ def _export_raw(fname, raw, physical_range, add_ch_type):
             if len(additional_patient_info) == 0:
                 additional_patient_info = None
             else:
-                additional_patient_info = " ".join(additional_patient_info)
+                # pyedflib does not support spaces in the patient name
+                additional_patient_info = "_".join(additional_patient_info)
 
             if birthday is not None:
-                if hdl.setPatientBirthDate(birthday[0], birthday[1], birthday[2]) != 0:
-                    raise RuntimeError(
-                        f"Setting patient birth date to {birthday} "
-                        f"returned an error"
-                    )
+                _try_to_set_value(hdl, "Birthdate", date(*birthday))
+
             for key, val in [
                 ("PatientCode", subj_info.get("his_id", "")),
                 ("PatientName", name),
-                ("PatientGender", sex),
-                ("AdditionalPatientInfo", additional_patient_info),
+                ("Gender", sex),
+                ("PatientAdditional", additional_patient_info),
```

Comment:
I don't think this matters for the function of the code, because the code is just using those to call a function (ex setGender, setPatientAdditional). I can change it though to be more consistent. @cbrnr

Reply:
You are right. Could you still change the order to be consistent with the specs?